Chinweezy/Banka-Andela-Developer-Challenge
# #165166072 setup server end of the application

[#### What does this PR do?
Connects to localhost port for testing endpoints on postman 

#### Description of Task to be completed?
Have app.js connect to server.js file

#### How should this be manually tested?
Run npm start:dev in the console of the project
"Server running on port 3000" should be seen
